client/my-sites/patterns/components/readymade-templates/section.tsx

```diff
@@ -66,12 +68,16 @@ export const ReadymadeTemplatesSection = ( {
 		return;
 	}
 
+	const props = forwardRef ? { forwardRef } : {};
+
 	return (
 		<PatternsSection
+			{ ...props }
```

Comment:
The previous discussion about this was solved but didn't get any answer to my last comment: https://github.com/Automattic/wp-calypso/pull/92794#discussion_r1684507904